examples/http-server/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require github.com/corazawaf/coraza/v3 v3.0.0-00010101000000-000000000000
```

Comment:
Currently the example can't be copy-pasted since it has this fake version number and replace directive. Most people will `go get` so it's not a huge deal, but recent Go does provide mechanisms that we can consider

https://go.dev/doc/tutorial/workspaces

If we have this sort of `go.work` we should be able to remove the replace here and make this a fully functional example even if it's copy-pasted out of the repo.
```
go 1.18

use (
	.
	./examples/http-server
)
```